app/assets/stylesheets/sitewide/sitewide.scss

```diff
@@ -217,11 +217,20 @@ body {
 }
 
 [class*="__cell"] {
-  flex: 1 0 auto;
+  flex: 0 0 auto;
```

Comment:
So... yeah this kinda screws up styles for matches page and the card page but I'll implement those with the new react table next so no need to worry about that 😉